gitoxide-core/src/repository/blame.rs

```diff
@@ -0,0 +1,62 @@
+use std::{ffi::OsStr, path::PathBuf, str::Lines};
+
+use anyhow::anyhow;
+use gix::bstr::BStr;
+
+pub fn blame_file(mut repo: gix::Repository, file: &OsStr, out: impl std::io::Write) -> anyhow::Result<()> {
+    repo.object_cache_size_if_unset(repo.compute_object_cache_size_for_tree_diffs(&**repo.index_or_empty()?));
+
+    let suspect = repo.head()?.peel_to_commit_in_place()?;
+    let traverse: Vec<_> =
+        gix::traverse::commit::topo::Builder::from_iters(&repo.objects, [suspect.id], None::<Vec<gix::ObjectId>>)
+            .build()?
+            .collect();
+    let mut resource_cache = repo.diff_resource_cache_for_tree_diff()?;
+
+    let work_dir: PathBuf = repo
+        .work_dir()
+        .ok_or_else(|| anyhow!("blame needs a workdir, but there is none"))?
+        .into();
+    let file_path: &BStr = gix::path::os_str_into_bstr(file)?;
+
+    let blame_entries = gix::blame::blame_file(
```

Comment:
@Byron This makes compiliation on CI fail as `gix::blame` is behind a feature flag. I feel like I’ve hit this issue already in the past, but I forgot how to resolve it. Would you be able to help me? :-)

Reply:
That's interesting - looks like the `blame` feature disappeared from the list somehow. In any case, it's back now and CI should get past this point (even though clippy has some legit complaints I locally at least).